Data_Structures_Answers.md

```diff
@@ -1,18 +1,25 @@
 Add your answers to the questions below.
 
-1. What is the runtime complexity of your ring buffer's `append` method?
+1. What is the runtime complexity of your ring buffer's `append` method? 
+O(1) complexity
 
-2. What is the space complexity of your ring buffer's `append` function?
+2. What is the space complexity of your ring buffer's `append` function? 
+O(1)
 
-3. What is the runtime complexity of your ring buffer's `get` method?
+3. What is the runtime complexity of your ring buffer's `get` method? 
+O(n)
 
-4. What is the space complexity of your ring buffer's `get` method?
+4. What is the space complexity of your ring buffer's `get` method? 
+O(n)
 
+5. What is the runtime complexity of the provided code in `names.py`? 
+O(n^2)
 
-5. What is the runtime complexity of the provided code in `names.py`?
+6. What is the space complexity of the provided code in `names.py`? 
+O(n^2)
```

Comment:
Hi Zech! I don't know what you talked about during your 1-1 but the space complexity for the provided code is O(1)